src/containers/Review/ReviewPage.tsx

```diff
@@ -268,4 +278,40 @@ const ApproveAllButton: React.FC<ApproveAllButtonProps> = ({
   )
 }
 
+interface PreviousStageDecisionProps {
+  review?: ReviewDetails | null
+  isFinalDecision: boolean
+  serial: string
+}
+
+const PreviousStageDecision: React.FC<PreviousStageDecisionProps> = ({
+  review,
+  isFinalDecision,
+  serial,
+}) =>
+  isFinalDecision && !!review ? (
+    <Segment.Group horizontal>
+      <Segment>
+        <Header as="h3">{strings.LABEL_PREVIOUS_REVIEW}:</Header>
+        <Link className="user-action" to={`/application/${serial}/review/${review.id}`}>
+          {strings.ACTION_VIEW}
+        </Link>
+        <ReviewByLabel user={review.reviewer} />
```

Comment:
Couple of suggestions:
- this is probably a good case for opening the link in a new tab? (So Consolidator doesn't lose their place)
- rather than "ViewReviewed by Consolidator 2" (which isn't really good English), maybe switch them round and add a separator, something like:  
   "Reviewed by Consolidator 2 | View"

Reply:
I've done an improvement for that - is now opening in a different tab, and the order of labels are changed. Hopefully that's better :)
<img width="750" alt="Screen Shot 2021-08-04 at 3 44 16 PM" src="https://user-images.githubusercontent.com/16461988/128118986-7d13d15c-0d5b-4f44-8ab0-5a17ea76ac77.png">

Reply:
Cool, thanks for that, much clearer. 
I personally prefer a seperator (either | or • usually) for these kind of things, but up to you if you can be bothered.

Eg.
<img width="750" alt="128118986-7d13d15c-0d5b-4f44-8ab0-5a17ea76ac77" src="https://user-images.githubusercontent.com/5456533/128286172-e466f3ef-23b0-4cc0-a86c-cdb2189d0838.png">

Reply:
Adding this now!